graylog2-server/src/main/java/org/graylog2/Configuration.java

```diff
@@ -134,6 +140,15 @@
     @Parameter(value = "dashboard_widget_default_cache_time", validator = PositiveDurationValidator.class)
     private Duration dashboardWidgetDefaultCacheTime = Duration.seconds(10L);
 
+    @Parameter(value = "content_packs_loader_enabled")
+    private boolean contentPacksLoaderEnabled = true;
+
+    @Parameter(value = "content_packs_dir", validators = DirectoryPathReadableValidator.class)
+    private Path contentPacksDir = Paths.get("data", "contentpacks");
+
+    @Parameter(value = "content_packs_auto_load", converter = TrimmedStringListConverter.class)
+    private List<String> contentPacksAutoLoad = Collections.emptyList();
```

Comment:
I think it might be better to make this a `Set<String>` to avoid potential problems with duplicates.